ptf/saimpls.py

```diff
@@ -3878,11 +3874,8 @@ def mplsEgressPhpTtlModeTest(self):
             eth_src=ROUTER_MAC,
             ip_dst='30.30.30.1',
             ip_ttl=63)
-        # Note: mpls_tag ttl is copied from top label, but it is not being
-        #       decremented even though it should be in uniform mode.
-        # Would we really pop labels but not swap what is left?
-        # Maybe this should change to swap with explicit null label?
-        mpls_tag['ttl'] = 56
+        
+        mpls_tag['ttl'] = 55
```

Comment:
Although 55 is correct from a standards point of view, that is not what we implement. This check should be removed.

Reply:
Yes, but we want to upstream the correct solution from the perspective of standards, not from our implementation

Reply:
@pkochmanska If we want to upstream the correct solution, we should setup a topology of switches in the test env, think about how routing protocols will distribute MPLS labels, come up with realistic scenarios of when multiple labels will be popped, and what is the right behavior for end-to-end packet forwarding. Every test case should be written for a specific network topology and should demonstrate push/pop/swap operations for various use cases like L3VPN, FRR, traffic engineering etc. We need to understand when/if a routing protocol will ever push a mpls route which pops multiple labels and also provide an IP nexthop.